### ISSUE TITLE
Starting games occasionally hangs.

### ISSUE THREAD

**Issue by curif** [curif/AgeOfJoy-2022.1#68]
Not allways, but some times...
```
 NullReferenceException: Object reference not set to an instance of an object.

01-11 09:39:13.627 17172 17204 E Unity   :   at LibretroMameCore.inputStateCB (System.UInt32 port, System.UInt32 device, System.UInt32 index, System.UInt32 id) [0x00000] in <00000000000000000000000000000000>:0

```

That happens when a game start, and not in a particular one.

**Comment by curif** [comment]
```
01-17 17:51:48.747 12876 12906 E CRASH   : backtrace:
01-17 17:51:48.747 12876 12906 E CRASH   :       #00 pc 00000000010af1dc  /data/app/com.curif.AgeOfJoy-oDwKn-Ze49HW9U3LMOe3Tg==/lib/arm64/mame2003_plus_libretro_android.so (BuildId: dfaf9ccc7ed3031bbd46211efcaea0c97fa70e4e)
01-17 17:51:48.747 12876 12906 E CRASH   :       #01 pc 000000000112c170  /data/app/com.curif.AgeOfJoy-oDwKn-Ze49HW9U3LMOe3Tg==/lib/arm64/mame2003_plus_libretro_android.so (BuildId: dfaf9ccc7ed3031bbd46211efcaea0c97fa70e4e)
01-17 17:51:48.747 12876 12906 E CRASH   :       #02 pc 00000000010ddc70  /data/app/com.curif.AgeOfJoy-oDwKn-Ze49HW9U3LMOe3Tg==/lib/arm64/mame2003_plus_libretro_android.so (BuildId: dfaf9ccc7ed3031bbd46211efcaea0c97fa70e4e)
01-17 17:51:48.747 12876 12906 E CRASH   :       #03 pc 000000000113f88c  /data/app/com.curif.AgeOfJoy-oDwKn-Ze49HW9U3LMOe3Tg==/lib/arm64/mame2003_plus_libretro_android.so (BuildId: dfaf9ccc7ed3031bbd46211efcaea0c97fa70e4e)
01-17 17:51:48.747 12876 12906 E CRASH   :       #04 pc 00000000010dbd34  /data/app/com.curif.AgeOfJoy-oDwKn-Ze49HW9U3LMOe3Tg==/lib/arm64/mame2003_plus_libretro_android.so (BuildId: dfaf9ccc7ed3031bbd46211efcaea0c97fa70e4e)
01-17 17:51:48.748 12876 12906 E CRASH   :       #05 pc 00000000010ab338  /data/app/com.curif.AgeOfJoy-oDwKn-Ze49HW9U3LMOe3Tg==/lib/arm64/mame2003_plus_libretro_android.so (retro_run+236) (BuildId: dfaf9ccc7ed3031bbd46211efcaea0c97fa70e4e)
01-17 17:51:48.748 12876 12906 E CRASH   :       #06 pc 0000000000d29fd4  /data/app/com.curif.AgeOfJoy-oDwKn-Ze49HW9U3LMOe3Tg==/lib/arm64/libil2cpp.so (BuildId: 2c0b9c231d6afb002fe0aaaebadd995f1f74522d)
01-17 17:51:48.748 12876 12906 E CRASH   :       #07 pc 0000000000d2c020  /data/app/com.curif.AgeOfJoy-oDwKn-Ze49HW9U3LMOe3Tg==/lib/arm64/libil2cpp.so (BuildId: 2c0b9c231d6afb002fe0aaaebadd995f1f74522d)
01-17 17:51:48.748 12876 12906 E CRASH   :       #08 pc 0000000000d2f734  /data/app/com.curif.AgeOfJoy-oDwKn-Ze49HW9U3LMOe3Tg==/lib/arm64/libil2cpp.so (BuildId: 2c0b9c231d6afb002fe0aaaebadd995f1f74522d)
01-17 17:51:48.748 12876 12906 E CRASH   :       #09 pc 0000000000980ccc  /data/app/com.curif.AgeOfJoy-oDwKn-Ze49HW9U3LMOe3Tg==/lib/arm64/libil2cpp.so (BuildId: 2c0b9c231d6afb002fe0aaaebadd995f1f74522d)
01-17 17:51:48.748 12876 12906 E CRASH   :       #10 pc 0000000000980bf0  /data/app/com.curif.AgeOfJoy-oDwKn-Ze49HW9U3LMOe3Tg==/lib/arm64/libil2cpp.so (BuildId: 2c0b9c231d6afb002fe0aaaebadd995f1f74522d)
01-17 17:51:48.748 12876 12906 E CRASH   :       #11 pc 00000000003df380  /data/app/com.curif.AgeOfJoy-oDwKn-Ze49HW9U3LMOe3Tg==/lib/arm64/libunity.so (scripting_method_invoke(ScriptingMethodPtr, ScriptingObjectPtr, ScriptingArguments&, Scr
iptingExceptionPtr*, bool)+164) (BuildId: 5ec529df357494b8e7cb2c7e57d4e924af684eec)
01-17 17:51:48.748 12876 12906 E CRASH   :       #12 pc 00000000003f0788  /data/app/com.curif.AgeOfJoy-oDwKn-Ze49HW9U3LMOe3Tg==/lib/arm64/libunity.so (ScriptingInvocation::Invoke(ScriptingExceptionPtr*, bool)+156) (BuildId: 5ec529df357494b
8e7cb2c7e57d4e924af684eec)
01-17 17:51:48.748 12876 12906 E CRASH   :       #13 pc 00000000003ff79c  /data/app/com.curif.AgeOfJoy-oDwKn-Ze49HW9U3LMOe3Tg==/lib/arm64/libunity.so (MonoBehaviour::CallUpdateMethod(int)+264) (BuildId: 5ec529df357494b8e7cb2c7e57d4e924af68
4eec)
01-17 17:51:48.748 12876 12906 E CRASH   :       #14 pc 00000000001eeb68  /data/app/com.curif.AgeOfJoy-oDwKn-Ze49HW9U3LMOe3Tg==/lib/arm64/libunity.so (void BaseBehaviourManager::CommonUpdate<BehaviourManager>()+216) (BuildId: 5ec529df35749
4b8e7cb2c7e57d4e924af684eec)
01-17 17:51:48.748 12876 12906 E CRASH   :       #15 pc 00000000001eea7c  /data/app/com.curif.AgeOfJoy-oDwKn-Ze49HW9U3LMOe3Tg==/lib/arm64/libunity.so (BehaviourManager::Update()+32) (BuildId: 5ec529df357494b8e7cb2c7e57d4e924af684eec)
01-17 17:51:48.748 12876 12906 E CRASH   :       #16 pc 00000000002a63a8  /data/app/com.curif.AgeOfJoy-oDwKn-Ze49HW9U3LMOe3Tg==/lib/arm64/libunity.so (BuildId: 5ec529df357494b8e7cb2c7e57d4e924af684eec)
01-17 17:51:48.748 12876 12906 E CRASH   :       #17 pc 00000000002a6a9c  /data/app/com.curif.AgeOfJoy-oDwKn-Ze49HW9U3LMOe3Tg==/lib/arm64/libunity.so (ExecutePlayerLoop(NativePlayerLoopSystem*)+92) (BuildId: 5ec529df357494b8e7cb2c7e57d4e92
4af684eec)
01-17 17:51:48.748 12876 12906 E CRASH   :       #18 pc 00000000002a6adc  /data/app/com.curif.AgeOfJoy-oDwKn-Ze49HW9U3LMOe3Tg==/lib/arm64/libunity.so (ExecutePlayerLoop(NativePlayerLoopSystem*)+156) (BuildId: 5ec529df357494b8e7cb2c7e57d4e9
24af684eec)
01-17 17:51:48.748 12876 12906 E CRASH   :       #19 pc 00000000002a6d88  /data/app/com.curif.AgeOfJoy-oDwKn-Ze49HW9U3LMOe3Tg==/lib/arm64/libunity.so (PlayerLoop()+324) (BuildId: 5ec529df357494b8e7cb2c7e57d4e924af684eec)
01-17 17:51:48.748 12876 12906 E CRASH   :       #20 pc 0000000000483e14  /data/app/com.curif.AgeOfJoy-oDwKn-Ze49HW9U3LMOe3Tg==/lib/arm64/libunity.so (UnityPlayerLoop()+748) (BuildId: 5ec529df357494b8e7cb2c7e57d4e924af684eec)
01-17 17:51:48.748 12876 12906 E CRASH   :       #21 pc 00000000004b90e4  /data/app/com.curif.AgeOfJoy-oDwKn-Ze49HW9U3LMOe3Tg==/lib/arm64/libunity.so (nativeRender(_JNIEnv*, _jobject*)+72) (BuildId: 5ec529df357494b8e7cb2c7e57d4e924af684eec
)
01-17 17:51:48.748 12876 12906 E CRASH   :       #22 pc 0000000000142950  /apex/com.android.runtime/lib64/libart.so (art_quick_alloc_array_resolved_region_tlab+112) (BuildId: 21503a17cc72c012bec3947a961f2d93)
01-17 17:51:48.804  1281 11396 I CompositorVR: SetSchedFifo( tid=12986, pol=2, pri=0 ) succeeded
01-17 17:51:48.891 12876 12906 E CRASH   : Tombstone written to: /storage/emulated/0/Android/data/com.curif.AgeOfJoy/files/tombstone_00
01-17 17:51:48.892 12876 12906 E AndroidRuntime: FATAL EXCEPTION: UnityMain
0
```

**Comment by curif** [comment]
Solved in v0.2.0
